galvan-support/src/main/java/org/terracotta/testing/rules/BasicExternalCluster.java

```diff
@@ -108,7 +108,19 @@ public BasicExternalCluster(File clusterDirectory, int stripeSize, List<File> se
 
   @Override
   public Statement apply(Statement base, Description description) {
```

Comment:
Unfortunately on my machine I have to drastically shorten the directory name.  This is the longest directory name that works for me "1234567890123456"  And this is the resulting path to the bat file:
C:\Users\ggib\dev\irepo\terracotta-enterprise\management\tms-system-tests\target\galvan\1234567890123456\testServer0\server\bin\start-tc-server.bat